src/components/useList/components/ListItemView/ListItemView.tsx

```diff
@@ -36,6 +36,10 @@ export interface ListItemViewProps extends QAProps {
      * Build in indentation component to render nested views structure
      */
     indentation?: number;
+    /**
+     * Show selected background if selected
+     */
+    hasSelectionBackground?: boolean;
```

Comment:
0c3ab33a5047d45574f4aafebe4c7c516c86f069

I don't want to do this through js, because it will be more difficult for other programmers to understand, and in the future, `select` may play a different role in the `ListItemView` and known will remember this js fix